packages/dapp-toolkit/src/wallet-request/transport/radix-connect-relay/radix-connect-relay-client.ts

```diff
@@ -0,0 +1,346 @@
+
+  const sessionChangeSubject = new Subject<Session>()
+
+  const deepLinkClient =
+    providers?.deepLinkClient ??
+    DeepLinkClient({
+      logger,
+      walletUrl,
+      callBackPath: '/connect',
+    })
+
+  const identityClient =
+    providers?.identityClient ??
+    IdentityClient({
+      providers: {
+        storageClient: storageClient.getPartition('identities'),
+        KeyPairClient: Curve25519,
+      },
+    })
+
+  const sessionClient =
+    providers?.sessionClient ??
+    SessionClient({
+      providers: {
+        storageClient: storageClient.getPartition('sessions'),
+        identityClient,
+      },
+    })
+
+  const radixConnectRelayApiClient = RadixConnectRelayApiClient({
+    baseUrl: `${baseUrl}/api/v1`,
+    logger,
+    providers: { encryptionClient },
+  })
+
+  const subscriptions = new Subscription()
+
+  const sendWalletLinkingRequest = (session: PendingSession) =>
+    identityClient
+      .get('dApp')
+      .mapErr(() => SdkError('FailedToReadIdentity', ''))
+      .andThen((dAppIdentity) =>
+        sessionClient
+          .patchSession(session.sessionId, { sentToWallet: true })
+          .mapErr(() => SdkError('FailedToUpdateSession', ''))
+          .andThen(() =>
+            deepLinkClient.deepLinkToWallet({
+              sessionId: session.sessionId,
+              origin,
+              publicKey: dAppIdentity.getPublicKey(),
+            }),
+          ),
+      )
+
+  const sendWalletInteractionRequest = (
+    session: ActiveSession,
+    walletInteraction: WalletInteraction,
+  ) =>
+    requestItemClient
+      .getById(walletInteraction.interactionId)
+      .mapErr(() =>
+        SdkError('FailedToGetPendingItems', walletInteraction.interactionId),
+      )
+      .andThen((pendingItem) =>
+        pendingItem
+          ? ok(pendingItem)
+          : err(
+              SdkError('PendingItemNotFound', walletInteraction.interactionId),
+            ),
+      )
+      .andThen((pendingItem) =>
+        requestItemClient
+          .patch(walletInteraction.interactionId, { sentToWallet: true })
+          .andThen(() =>
+            radixConnectRelayApiClient.sendRequest(
+              session,
+              pendingItem.walletInteraction,
+            ),
+          )
+          .andThen(() =>
+            deepLinkClient.deepLinkToWallet({
+              sessionId: session.sessionId,
+              interactionId: pendingItem.interactionId,
+            }),
+          ),
+      )
+      .mapErr(() =>
+        SdkError('FailedToSendDappRequest', walletInteraction.interactionId),
+      )
+
+  const sendToWallet = (
+    walletInteraction: WalletInteraction,
+    callbackFns: Partial<CallbackFns>,
+  ): ResultAsync<WalletInteractionResponse, SdkError> =>
+    sessionClient
+      .getCurrentSession()
+      .mapErr(() =>
+        SdkError('FailedToReadSession', walletInteraction.interactionId),
+      )
+      .andThen((session) =>
+        (session.status === 'Pending'
```

Comment:
why not enum/const value for `Pending`?

Reply:
because status has type `status: "Pending" | "Active"`